src/app/core/services/session/session-api.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { Session } from '../../state/session/session.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SessionApiService {
```

Comment:
Please move this file to another folder to not confuse with the other services. I recommend having a folder `http` or `api` in the `core` folder and putting all the API related services inside.

Reply:
Done